src/card/card-actions.jsx

```diff
@@ -24,8 +24,10 @@ let CardActions = React.createClass({
       });
     });
 
+    let mergedStyles = this.mergeAndPrefix(styles.root, this.props.style);
```

Comment:
mergedStyles will not be changed here. I would say that other components should also use const.
If you want to learn more about it : https://hacks.mozilla.org/2015/07/es6-in-depth-let-and-const.